test/runtests.jl

```diff
@@ -461,3 +461,45 @@ end
     pa[:] = [1 2; 3 4]
     @test pa == [1 2; 3 4]
 end
+
+@testset "repeat" begin
+    pa1 = PooledArray([1, 2, 3])
+    pa2 = repeat(pa1)
+    pa3 = repeat(pa1, 2)
+    pa4 = repeat(pa1, inner = 2)
+    @test pa2 == pa1
+    @test pa3 == [1, 2, 3, 1, 2, 3]
+    @test pa4 == [1, 1, 2, 2, 3, 3]
+    @test refpool(pa1) === refpool(pa2) === refpool(pa3) === refpool(pa4)
+    @test invrefpool(pa1) === invrefpool(pa2) === invrefpool(pa3) === invrefpool(pa4)
+    @test refcount(pa1) === refcount(pa2) === refcount(pa3) === refcount(pa4)
+    @test refcount(pa1)[] == 4
+
+    pa1 = PooledArray(["one", "two"])
+    # check it didn't mess the other pa
+    @test pa3 == [1, 2, 3, 1, 2, 3]
+
```

Comment:
This doesn't check much AFAICT: no significant operation has been performed since the last check.
```suggestion
    pa1 = PooledArray(["one", "two"])
```

Reply:
Should I remove it?

Reply:
What's what the suggestion does?

Reply:
Sorry, I am lost. What is your suggestion on lines 478-480?

Reply:
Not sure why GitHub is confused about it, but I just suggested dropping lines 479-481.

Reply:
I see,
GitHub doesn't allow to commit suggestion (the suggestion is invalid...)!

Reply:
done!